### ISSUE TITLE
admincli is very broken

### ISSUE THREAD

**Issue by GTANAdam** [chenhg5/go-admin#15]
I've been having some headaches with generating templates and I'm honestly not having fun.. I will have some commits pushed to get admincli generating good templates..

**Comment by chenhg5** [comment]
@GTANAdam yes, the cli tool has not been fully tested and may have some unexpected problems. You can help describe them here and see how can we improve it. PR is also very welcome. Thx.

**Comment by theromis** [comment]
fixed some of them here https://github.com/chenhg5/go-admin/pull/22

**Comment by chenhg5** [comment]
add interactive prompts support [#a8a9a3](https://github.com/chenhg5/go-admin/commit/a8a9a358e71c12fea412777a9f3380b5dd48a378)
